getndazn/dazn-lambda-powertools
# Move to DAZN npm org

## What did you implement:

Move everything from `@perform` org on npm to `@dazn`



## How did you implement it:



## How can we verify it:

Install the latest version numbers from the `@dazn` org - see https://www.npmjs.com/package/@dazn/lambda-powertools-logger etc



## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
